phpBB/phpbb/db/migration/schema_generator.php

```diff
@@ -74,7 +88,11 @@ public function __construct(array $class_names, \phpbb\config\config $config, \p
 	/**
 	* Loads all migrations and their application state from the database.
 	*
-	* @return array
+	* @return array An array describing the database schema.
+	*
+	* @throws UnexpectedValueException  If a migration tries to use an undefined schema change.
+	* @throws UnexpectedValueException	If a dependency can't be resolved or there are circular
+	* 									dependencies between migrations.
 	*/
 	public function get_schema()
```

Comment:
While you're on it, maybe change it to this:
```suggestion
	public function get_schema() : array
```
Would you mind maybe adding the return type hint to all methods in this class?

Reply:
Will do